models/stg_pardot__list_email.sql

```diff
@@ -251,6 +257,74 @@ list_emails_enhanced as (
     
     from list_emails_joined
 
+),
+
+global_list_emails_standard as ( -- An email specific URL builder was rolled out globally in late FY25, with full adoption across markets from FY26. See https://theirc.github.io/URLBuilder/#emailUrlGenPage
+    
+    select
+        *,
+        /* Parsing part 2 of the list email name into component parts to enable filters/group bys on Power BI reports
+        
+        Example Dec01MLM needs to be broken up further to show month_abbreviation (Dec), email version number (01) and email segment code (MLM) */
+
+        /* Parsing email month_abbreviated */
+        case 
+            when list_email_name_part_1 ilike 'FY%'
+            and character_length(list_email_name_part_2) in (8,9)
+            then left(list_email_name_part_2,3)
+        else null
+        end as email_url_builder_month_abbreviated,
```

Comment:
```suggestion
        end as list_email_url_builder_month_abbreviated,
```

I think these should be modified to 'list_email' here, since at this level they are 'list_email' attributes. In the pardot schema an 'email' is a different object, representing an individual email send to a prospect (which is also represented by the corresponded `visitor_activity`)